### PR TITLE
LPS-154530 Improvement of margins-paddings component

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/SpacingBox.js [hunk 1]
```diff
@@ -164,6 +174,15 @@ function SpacingSelectorButton({field, onChange, position, type, value}) {
 		}
 	}, [active]);
 
+	const spacingOptionValue = useCallback(
+		(optionValue) => {
+			const value = tokenValues[`spacer${optionValue}`].value;
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/SpacingBox.js:185]: How this works in a site without tokens? I think in this case we should show the computed value

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**